amazon-msk/cdktf/web-streaming/zilla.yaml.mustache

```diff
@@ -0,0 +1,210 @@
+name: {{name}}
+vaults:
+  secure:
+    type: aws-secrets
+{{#glueEnabled}}
+catalogs:
+  glue_catalog:
+    type: aws-glue
+    options:
+      registry: {{glueRegistry}}
+{{/glueEnabled}}
+telemetry:
+  metrics:
+    - stream.active.received
+    - stream.active.sent
+    - stream.opens.received
+    - stream.opens.sent
+    - stream.data.received
+    - stream.data.sent
+    - stream.errors.received
+    - stream.errors.sent
+    - stream.closes.received
+    - stream.closes.sent
+  exporters:
+    stdout_logs_exporter:
+      type: stdout
+{{#cloudwatch}}
+    aws0:
+      type: aws-cloudwatch
+      options:
+        logs:
+          group: {{cloudwatch.logs.group}}
+          stream: events
+        metrics:
+          namespace: {{cloudwatch.metrics.namespace}}
+{{/cloudwatch}}
+bindings:
+  tcp_server:
+    type: tcp
+    kind: server
+    options:
+      host: 0.0.0.0
+      port: {{publicPort}}
+    telemetry:
+      metrics:
+        - stream.*
+    exit: tls_server
+  tls_server:
+    type: tls
+    kind: server
+    vault: secure
+    options:
+      keys:
+        - {{publicTlsCertificateKey}}
+    telemetry:
+      metrics:
+        - stream.*
+    exit: http_server
+  http_server:
+    type: http
+    kind: server
+    telemetry:
+      metrics:
+        - stream.*
+    routes:
+      - when:
+          - headers:
+              :path: /streams{{{path}}}
```

Comment:
Why do we force the streams prefix? Is it to disambiguate with other CRUD operations?
If so, let's instead keep the same path and route by `accept` header with value `text/event-stream` as that is the required content type for SSE response.